test/e2e/e2e_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
```

Comment:
@wewang58 I think, we can add https://github.com/openshift/lws-operator/pull/38 in here too.

Reply:
@ardaguclu Yes, opened a pr: https://github.com/openshift/jobset-operator/pull/9